### PR TITLE
Pfm 2155

### PR DESCRIPTION
**Pull request by vguptatw**
Request you to please review it and take appropiate action. Changes related to : 
https://digit-discuss.atlassian.net/browse/PFM-2155

### CODE REVIEW

File: backend/attendance-service/src/main/java/org/egov/util/AttendanceServiceConstants.java [hunk 1]
```diff
@@ -3,12 +3,12 @@
 public class AttendanceServiceConstants {
     public static final String MASTER_TENANTS = "tenants";
```
Comment (subhashini-egov) [inline backend/attendance-service/src/main/java/org/egov/util/AttendanceServiceConstants.java:4]: @vguptatw - shouldn't we get roles from roles.json in MDMS? The risk here is that if the role codes change there, we will have to keep updating here.

**Review by subhashini-egov — CHANGES_REQUESTED** [review]
I have just one comment about the roles constants. Can you please comment on that @vguptatw ?

**Comment by vguptatw** [comment]
Hi @subhashini-egov , we have updated Register search API. Now it is configuration driven. If role is not superuser, then we restrict it to the tenant and registers this user is enrolled in. 

Request you to please review the changes.

**Review by subhashini-egov — APPROVED** [review]
Reviewed and approved.

**Pull request merged**